bin/adhoc-scripts/fixWorkflowParentage.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
This may or may not be relevant, but since we rely on python3 I think it should be python3 instead of python. The later can be pointed to something else depending on node configuration and software setup.

Reply:
If I am not wrong, distutils automatically converts that to `python3`, if it is using a python3 stack to build the service.